concent_api/core/tasks.py

```diff
@@ -67,9 +68,11 @@ def upload_finished(subtask_id: str):
             return
 
         # Change subtask state to ADDITIONAL VERIFICATION.
-        subtask.state = Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name  # pylint: disable=no-member
-        subtask.full_clean()
-        subtask.save()
+        update_subtask_state(
+            subtask=subtask,
+            state=Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name,  # pylint: disable=no-member
+            next_deadline=subtask.next_deadline.timestamp() + settings.SUBTASK_VERIFICATION_TIME
```

Comment:
as the function accepts `Optional[int]` and here we are adding `float + int`, I think I would like to have casting to `int`

Reply:
done